src/@next/Select/components/Activator/ActivatorSelect.tsx

```diff
@@ -76,13 +77,14 @@ export const ActivatorSelect = ({
     if (hasSelectedValues) {
       return (
         <TagsContainer>
-          <StyledTag
+          <Tag
+            style={{ width: 28, marginRight: space8 }}
```

Comment:
somehow it will have error rendering this page if we use styled component of `Tag` in this page

so i have to not use styled component instead assign a style like the above to the tag

Reply:
Hmmm can we investigate this more? I don't think we should update the implementations to cater storybook upgrade